implementation/src/main/java/io/smallrye/config/ConfigValueConfigSourceWrapper.java

```diff
@@ -0,0 +1,70 @@
+package io.smallrye.config;
+
+import java.io.Serializable;
+import java.util.Map;
+import java.util.Set;
+
+import org.eclipse.microprofile.config.spi.ConfigSource;
+
+public class ConfigValueConfigSourceWrapper implements ConfigValueConfigSource, Serializable {
```

Comment:
This could probably be `final` and non-`public`, could it not?

Reply:
Yes.